betamax/adapter.py

```diff
@@ -125,6 +166,7 @@ def find_adapter(self, url):
 
 
 def unhandled_request_message(request, cassette):
+    """Exception for unhandled requests."""
```

Comment:
Docstrings should start with an imperative, e.g., "Generate exception message for unhandled requests."